public/jsx/components/input-error.jsx

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var React = require('react');
+
+var InputError = React.createClass({
+  displayName: 'InputError',
+
+  propTypes: {
+    optModifier: React.PropTypes.oneOf(['center', 'right', 'left']),
+    text: React.PropTypes.string.isRequired,
+  },
+
+  render: function() {
+    var { text, ...toolTipAttrs } = this.props;
+
+    var errorClassList = ['tooltip'];
+    errorClassList.push(this.props.optModifier || 'left');
+    var errorClass = errorClassList.join(' ');
```

Comment:
Could you use classSet here? https://facebook.github.io/react/docs/class-name-manipulation.html